frontend/components/ui/widget.tsx

```diff
@@ -0,0 +1,70 @@
+export const Container = (
+  props: React.PropsWithChildren<{ className?: string }>
+): JSX.Element => (
+  <div
+    className={`
+      !block !mx-0 !py-8 px-4 rounded-md
```

Comment:
this classname syntax is wild whats going on here

Reply:
- block is display:block
- mx-0 is margin-left: 0; margin-right: 0;
- py-8 is padding-top: 8rem; padding-bottom: 8rem;
- px-4 is padding-left: 4rem; padding-right: 4rem;
- rounded-md is border-radius medium
All the `!` are to make them `!important` because the index.css messes up with everything with high-specificity selectors.